login.php

```diff
@@ -63,7 +63,7 @@
           <input type="password" class="form-control" name="password" required placeholder="Password">
           <div class="input-group-append">
             <div class="input-group-text">
-              <span class="fas fa-lock"></span>
+              <span class="fas fa-eye"></span>
```

Comment:
I dont see the `password-toggle` added here - Where will the function come? Is there any onClick handler you wanted to add?

Reply:
Hi sorry I've made an update to add an id. Please check it out.